benchmark/common.js

```diff
@@ -3,6 +3,8 @@
 const child_process = require('child_process');
 const http_benchmarkers = require('./_http-benchmarkers.js');
 
+exports.buildType = process.config.target_defaults.default_configuration;
```

Comment:
So this is not 100% correct, it indicated who `./configure` was run, not how the binary was built.
For every `./configure` run we get two sets of build files, one that generates a `debug` binary and one  for `release`.
For make builds the binary depends on the `BUILDTYPE` env var (as referenced above).
In MSBuild it depends on `/t:%target%` var:
https://github.com/nodejs/node/blob/74ba48294b115364b92bbd5c87c8025cd109e303/vcbuild.bat#L304

tl;dr I don't have a solution, except to `try` both types in the files below:
```js
try {
  v8 = require('./build/Release/binding');
} catch (err) {
  try {
    v8 = require('./build/Debug/binding');
  } catch (err) {
    console.error(`${__filename}: V8 Binding failed to load`);
    process.exit(0);
  }
}
```

Reply:
NM, found it
```suggestion
exports.buildType = process.features.debug
```

Reply:
Ah nice, I was not aware of that. Thanks!